### PR TITLE
AP-1350 Update GET user, filter user and transaction response for US

### PR DESCRIPTION
**Pull request by shaidarazum**
Update following endpoint response based on change for US customers : 
- GET user, 
- Filter user 
- Create transaction

Task Jira
https://zum-application.atlassian.net/jira/software/c/projects/AP/boards/1?modal=detail&selectedIssue=AP-1350

Add US postman collection
https://zum-application.atlassian.net/jira/software/c/projects/AP/boards/1?modal=detail&selectedIssue=AP-1355

### CODE REVIEW

File: docs/us/api-specification/transactions.mdx [hunk 1]
```diff
@@ -165,9 +173,50 @@ values={[
...
+| Id                          | guid     | The wallet id                                                                                              |
+| Type                        | string   | The wallet type                                                                                            |
+| **FundingSource**           |          | If transaction has a funding source \* Not all information from a user is returned in this endpoint        |
+| Id                          | guid     | The funding source id                                                                                      |
+| Institution                 | string   | The institution name                                                                                       |
+| InstitutionNumber           | string   | The institution number                                                                                     |
+| TransitNumber               | string   | The transit number                                                                                         |
```
Comment (krishna-vempati) [inline docs/us/api-specification/transactions.mdx:211]: do we return `TransitNumber` and `Institution` for US transactions?

File: docs/us/api-specification/transactions.mdx [hunk 2]
```diff
@@ -165,9 +173,50 @@ values={[
...
+| CreatedAt                   | datetime | When the transaction was created                                                                           |
+| Memo                        | string   | Transaction memo                                                                                           |
+| Comment                     | string   | Transaction comment                                                                                        |
+| Amount                      | decimal  | Transaction amount                                                                                         |
+| Currency                    | string   | Transaction currency                                                                                       |
+| ZumRailsType                | string   | Transaction type                                                                                           |
+| TransactionStatus           | string   | Indicate the status of the transaction                                                                     |
```
Comment (guilhermeshiba) [inline docs/us/api-specification/transactions.mdx:185]: shouldn't it be "indicates"? ignore if not

File: docs/us/api-specification/transactions.mdx [hunk 3]
```diff
@@ -165,9 +173,50 @@ values={[
...
+| FailedTransactionEvent      | string   | If the transaction has failed, the [transaction event](#status-and-events) that caused it (null otherwise) |
+| ScheduledStartDate          | date     | The date transaction will be sent to the financial institution                                             |
```
Comment (guilhermeshiba) [inline docs/us/api-specification/transactions.mdx:187]: The date **that the** transaction will be sent to the financial institution ?

File: docs/us/api-specification/transactions.mdx [hunk 4]
```diff
@@ -165,9 +173,50 @@ values={[
...
+| RoutingNumber               | string   | The account number                                                                                         |
+| **TargetWallet**            |          | If transaction has a target wallet                                                                         |
+| Id                          | guid     | The target wallet id                                                                                       |
+| Type                        | string   | The target wallet type                                                                                     |
+| **TransactionHistory**      |          | List of transaction history events                                                                         |
+| CreatedAt                   | datetime | When the transaction event happened                                                                        |
+| Event                       | string   | The event happened                                                                                         |
```
Comment (guilhermeshiba) [inline docs/us/api-specification/transactions.mdx:218]: The event **that** happened

**Review by guilhermeshiba — APPROVED** [review]

File: docs/us/api-specification/transactions.mdx (hunk 2 above)
Comment (shaidarazum) [reply docs/us/api-specification/transactions.mdx]: Corrected !

File: docs/us/api-specification/transactions.mdx (hunk 1 above)
Comment (shaidarazum) [reply docs/us/api-specification/transactions.mdx:211]: Yes seems like we do
transaction_us_hide_fieds

**Pull request merged**